identitynow.sgnl.yaml

```diff
@@ -0,0 +1,239 @@
+# IdentityNow Configuration YAML for DS2.0
+displayName: "IdentityNow"
+icon: "PHN2ZyB3aWR0aD0iNjQiIGhlaWdodD0iNjQiIHZpZXdCb3g9IjAgMCA2NCA2NCIgZmlsbD0ibm9uZSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KPHBhdGggZD0iTTQwLjQxMTggNDIuMTM3M0gxMEwzNS4yMzUzIDEwTDQwLjQxMTggNDIuMTM3M1oiIGZpbGw9IiMwMDMzQTEiLz4KPHBhdGggZD0iTTQwLjQxMTggNDIuMTM3M0g1NEwzNS4yMzUzIDEwTDQwLjQxMTggNDIuMTM3M1oiIGZpbGw9IiNDQzI3QjAiLz4KPHBhdGggZD0iTTQwLjQxMTggNDIuMTM3M0gxMEw0Mi4zNTI5IDUzLjc4NDNMNDAuNDExOCA0Mi4xMzczWiIgZmlsbD0iIzAwNzFDRSIvPgo8cGF0aCBkPSJNNDAuNDExOCA0Mi4xMzczSDU0TDQyLjM1MjkgNTMuNzg0M0w0MC40MTE4IDQyLjEzNzNaIiBmaWxsPSIjRTE3RkQyIi8+Cjwvc3ZnPgo="
+description: "IdentityNow as a System of Record"
+# Address does not always need to end with identitynow.com suffix. For example, identitynow-demo.com.
+address: "{{Input Required: IdentityNow-Tenant}}.api.identitynow.com"
+defaultSyncFrequency: HOURLY
+defaultSyncMinInterval: 1
+defaultApiCallFrequency: SECONDLY
+defaultApiCallMinInterval: 1
+type: "IdentityNow-1.0.0"
+# Example Config:
+# {
+#   "apiVersion": "v3",
+#   "entityConfig": {
+#     "accounts": {
+#       "uniqueIDAttribute": "id"
+#     },
+#     "entitlements": {
+#       "uniqueIDAttribute": "id",
+#       "apiVersion": "beta" // overrides the top level apiVersion.
+#     },
+#     "accountEntitlements": {
+#       "uniqueIDAttribute": "id",
+#       "apiVersion": "beta"
+#     },
+#     "identities": {
+#       "uniqueIDAttribute": "id",
+#       "apiVersion": "beta"
+#     }
+#   }
+# }
+adapterConfig: "ewogICJhcGlWZXJzaW9uIjogInYzIiwKICAiZW50aXR5Q29uZmlnIjogewogICAgImFjY291bnRzIjogewogICAgICAidW5pcXVlSURBdHRyaWJ1dGUiOiAiaWQiCiAgICB9LAogICAgImVudGl0bGVtZW50cyI6IHsKICAgICAgInVuaXF1ZUlEQXR0cmlidXRlIjogImlkIiwKICAgICAgImFwaVZlcnNpb24iOiAiYmV0YSIKICAgIH0sCiAgICAiYWNjb3VudEVudGl0bGVtZW50cyI6IHsKICAgICAgInVuaXF1ZUlEQXR0cmlidXRlIjogImlkIiwKICAgICAgImFwaVZlcnNpb24iOiAiYmV0YSIKICAgIH0sCiAgICAiaWRlbnRpdGllcyI6IHsKICAgICAgInVuaXF1ZUlEQXR0cmlidXRlIjogImlkIiwKICAgICAgImFwaVZlcnNpb24iOiAiYmV0YSIKICAgIH0KICB9Cn0K"
+# All auth mechanisms specified below must be supported by the specified Adapter.
+auth:
+  - oAuth2ClientCredentials:
+      clientId: "{{Input Required: Client-ID}}"
+      tokenUrl: "https://{{Input Required: IdentityNow-Domain}}/oauth/token"
+      authStyle: AutoDetect
+      clientSecret: "{{Input Required: Client-Secret}}"
+entities:
+  Account:
+    displayName: Account
+    externalId: accounts
+    description: Account entity in IdentityNow
+    syncFrequency: HOURLY
+    syncMinInterval: 1
+    apiCallFrequency: SECONDLY
+    apiCallMinInterval: 1
+    pageSize: 250
```

Comment:
why 250?

Reply:
default and max page size of Sailpoint IDN - https://developer.sailpoint.com/idn/api/v3/list-accounts